libs/blocks/section-metadata/section-metadata.css

```diff
@@ -137,6 +137,46 @@
   gap: var(--spacing-xxxl);
 }
 
+.section.border-radius-block-s > div,
+.section.border-radius-block-s > div .background picture img {
+  border-radius: var(--border-radius-s);
+}
+
+.section.border-radius-block-l > div,
+.section.border-radius-block-l > div .background picture img {
+  border-radius: var(--border-radius-l);
+}
+
+.section.border-radius-block-xl > div,
+.section.border-radius-block-xl > div .background picture img {
+  border-radius: var(--border-radius-xl);
+}
+
+.section.no-border-block-radius > div,
+.section.no-border-block-radius > div .background picture img {
+  border-radius: var(--no-border-radius) !important;
```

Comment:
Why is the `!important` needed? Same question for L173. What other rule might override this one?

Reply:
Removed it. There is no rule to remove it right now.